### PR TITLE
Add Freeplane 1.7.3

### PR DESCRIPTION
**Pull request by maanur**

### CODE REVIEW

File: freeplane.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "homepage": "http://freeplane.sourceforge.net/wiki/index.php/Main_Page",
```
Comment (r15ch13) [inline freeplane.json]: Change `homepage` to https://www.freeplane.org

File: freeplane.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "version": "1.7.3",
+    "hash": "sha1:30e3893d1855adb4db44f8f3ff74150c6476035c",
+    "description": "Freeplane is a free and open source software application that supports thinking, sharing information and getting things done at work, in school and at home. The software can be used for mind mapping and analyzing the information contained in mind maps.",
+    "license": "GNU GPL V2+",
```
Comment (r15ch13) [inline freeplane.json]: Change `license` to `GPL-2.0-or-later`

File: freeplane.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            "java/openjdk"
+        ]
+    },
+    "shortcuts": [
+        [
+            "freeplane.exe",
+            "Freeplane"
+        ]
+    ],
+    "checkver": {
```
Comment (r15ch13) [inline freeplane.json]: Could be changed to:

```json
    "checkver": {
        "url": "https://www.freeplane.org/info/history/history_en.txt",
        "re": "=+\\s([\\d.]+)\\s=+"
    },
```

Comment (maanur) [reply freeplane.json]: Changed regex a bit, I hope it's ok now.

**Pull request merged**